weblate/api/serializers.py

```diff
@@ -177,10 +178,7 @@ def get_value(self, dictionary):
 
 class FullUserSerializer(serializers.ModelSerializer):
     groups = serializers.HyperlinkedIdentityField(
-        view_name="api:group-detail",
-        lookup_field="id",
-        many=True,
-        read_only=True,
+        view_name="api:group-detail", lookup_field="id", many=True, read_only=True
```

Comment:
Reverting change caused by mixed up black versions.

```suggestion
        view_name="api:group-detail",
        lookup_field="id",
        many=True,
        read_only=True,
```